app/views/events/_form.html.erb

```diff
@@ -1,67 +1,98 @@
+      <div class="col-lg-10">
+        <%= f.select(:game_mode, Tournament.game_modes.map {|k, v| [k.humanize, v]},
+                     {:include_blank => '----'},
+                     { :class => 'form-control', :id => 'event_game_mode_tournament'}) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.player_type'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.select(:player_type, Event.player_types.map {|k, v| [k.humanize, v]},
+                     {:include_blank => '----'},
+                     { :class => 'form-control'}) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.max_teams'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.text_field :max_teams, id: :event_max_teams, :class => 'form-control' %>
+        <%= f.error_span(:max_teams) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.discipline'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.text_field :discipline, id: :event_discipline, :class => 'form-control' %>
+        <%= f.error_span(:discipline) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.deadline'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.text_field :deadline, id: :event_deadline, :data => { :provide => 'datepicker', "date-format" => "dd/mm/yyyy" }, :class => 'form-control' %>
+        <%= f.error_span(:deadline) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.startdate'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.text_field :startdate, id: :event_startdate, :data => {:provide => 'datepicker', "date-format" => "dd/mm/yyyy"}, :class => 'form-control' %>
+        <%= f.error_span(:startdate) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.enddate'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.text_field :enddate, id: :event_enddate, :data => {:provide => 'datepicker', "date-format" => "dd/mm/yyyy"}, :class => 'form-control' %>
+        <%= f.error_span(:enddate) %>
+      </div>
+    </div>
+    <div class="form-group">
+      <%= f.label t('events.new.duration'), :class => 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.text_field :duration, id: :event_duration, :disabled => true, :class => 'form-control' %></div>
+    </div>
+    <div class="form-group">
+      <div class="col-lg-offset-2 col-lg-10">
+        <%= f.submit t('events.create_event'), :class => 'btn btn-primary' %>
+        <%= link_to t('.cancel', :default => t("helpers.links.cancel")),
+                    events_path, :class => 'btn btn-default' %>
+      </div>
+    </div>
```

Comment:
this all looks fine to me but @erksch has way more knowledge about front end development than me so he should probably have a look at this.